champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/data/BlockTossObject.java

```diff
@@ -269,14 +273,21 @@ private void playCues() {
     }
 
     // can be run out of main thread
-    public void impact(Player caster) {
+    public void impact(Player caster, LivingEntity target) {
         if (impacted) {
             return;
         }
 
+        boolean alreadyHit = false;
+
         this.impacted = true;
         final Location impactLocation = getCenterLocation();
 
+        if (target != null) {
+            doDamage(impactLocation, target, caster, true);
+            alreadyHit = true;
```

Comment:
This is going to cause the attacker players to not receive feedback chat messages, can you verify this?

Reply:
yes because I forgot to add them to the damaged map